tests/python/tvmscript/test_tvmscript_parser_tir.py

```diff
@@ -272,7 +272,7 @@ def test_tir_starred_for_loop():
     @T.prim_func(private=True)
     def starred(a: T.handle, b: T.handle):
         A = T.match_buffer(a, [*dims, 128], "int32")
-        B = T.match_buffer(a, dims, "int32")
+        B = T.match_buffer(b, dims, "int32")
```

Comment:
This caused a well-formed error but I'm pretty sure it was a bug in the first place.

Reply:
Agreed, I think this was a bug in the unit test.